src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java

```diff
@@ -150,12 +153,12 @@ public boolean canCommandSenderUseCommand(Commands theCommand)
     @NotNull
     public boolean isPlayerOpped(@NotNull final ICommandSender sender, String cmdName)
     {
-        int requiredOpLevel = PERMNUM;
-        if (PERMNUM < 1)
+        int requiredOpLevel = Configurations.opLevelForServer;
+        if (Configurations.opLevelForServer < 1)
         {
             requiredOpLevel = 1;
         }
-        if (PERMNUM > 4)
+        if (Configurations.opLevelForServer > 4)
```

Comment:
I did this in the event that the number the user puts into the config is out of range, it will auto adjust it so as not to throw an error